homeassistant/components/media_source/strings.json

```
@@ -9,5 +9,8 @@
     "unknown_media_source": {
       "message": "Unknown media source: {domain}"
     }
+  },
+  "common": {
+    "sources_default": "Media sources"
```

```suggestion
    "sources_default": "Media Sources"
```

At least this is what it has been.

---

But it should be sentence-cased now to fix how it looks at the moment.
We will be translating "sources" here so it becomes "Medienquellen" in German or "Sources média" in French.